website/docs_issues.qmd

```diff
@@ -23,26 +23,44 @@ The only exception are .Rd-files for data sets, marked with the `\docType{data}`
 
 Sometimes functions don't return one specific value but are rather called for their side effects. In that case the `\value`-tag should state this.
 
-```{r, eval=FALSE}
+```default
 \value{No return value, called for side effects}
 ```
 
 When using 'roxygen' to render the .Rd-files, an `@return`-tag must be added in the corresponding .R-file. This will create the `\value`-tag automatically when rendering.
 
-```{r}
+```default
 #' @return What your function returns.
 ```
 
-For more details on 'roxygen2' check the 'roxygen2' section.
+For more details on 'roxygen2' check the ['roxygen2' section](#repeated-rejections-of-issues-in-manuals-if-using-roxygen2).
 
 
 
-# Issue 2
+
+# Repeated Rejections of Issues in Manuals If Using 'roxygen2' {#repeated-rejections-of-issues-in-manuals-if-using-roxygen2}
 
 ## Problem
 
+The CRAN team rejected your package for issues regarding the manuals a second time, even if you already changed them according to their suggestions.
+
 ## Solution
 
+Implement your changes in the corresponding .R-files instead of the .Rd-files. Before resubmitting render the .Rd-files again using `roxygenize()`.
```

Comment:
It might be useful to also mention the use of the [`document()`](https://devtools.r-lib.org/reference/document.html) function and I think lots of folks might be using 'devtools' 📦

Reply:
Good point! I will look more into this and add a paragraph after uesR!2024.